dashboard/test/models/level_test.rb

```diff
@@ -274,12 +274,6 @@ def raise_unless_specifies_ideal_level_source(level_class)
     assert_equal time, level.updated_at.to_i
   end
 
-  test 'update_ideal_level_source does nothing for maze levels' do
```

Comment:
Removed this test because I have Maze levels in my database that have ideal level source so it seemed no longer necessary.